api/swagger-spec/oapi-v1.json

```diff
@@ -29196,6 +29196,10 @@
      "wildcardPolicy": {
       "type": "string",
       "description": "Wildcard policy is the wildcard policy that was allowed where this route is exposed."
+     },
+     "routerCanonicalHostname": {
+      "type": "string",
+      "description": "CanonicalHostname is the external host name for the router that can be used as a CNAME for the host requested for this route. This value is optional and may not be set in all cases."
```

Comment:
@knobunc The first form states the capability. The second form adds that you can do it if you have permission. Alternatively, if you want it done, go find someone with permission to do it. This leads back to the original position (from a while ago): talk to the person that can do it.

I personally don't see much improvement in the second wording.